Pipelines/Templates/build-Solution.yml

```diff
@@ -50,42 +50,58 @@ steps:
   displayName: 'Flatten JSON files'
   enabled: true
   
-- powershell: |
-    $manifestVersion = ''
-    $manifestPaths = Get-ChildItem -Path "$(Build.SourcesDirectory)\$(RepoName)\$(SolutionName)" -Recurse -Filter *CanvasManifest.json
-    foreach($manifestPath in $manifestPaths) {
-        $manifest = Get-Content $manifestPath.FullName | ConvertFrom-Json
-        if($manifest.FormatVersion -ne '') {
-            $manifestVersion = $manifest.FormatVersion
-            break
-        }
-    }
-    Write-Host "##vso[task.setvariable variable=ManifestVersion]$manifestVersion"
-    Write-Host "Version: $manifestVersion"
-  displayName: 'Get Unpacked Version'
-  condition: succeeded()
+# - powershell: |

```

Comment:
Will address this one.

Reply:
Actually, reconsidering since there is a lot of powershell vs pwsh tasks.  I am creating a separate enhancement to address this.
https://github.com/microsoft/coe-starter-kit/issues/3041